com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -472,7 +494,13 @@ public HDRenderPipeline(HDRenderPipelineAsset asset, HDRenderPipelineAsset defau
             // Propagate it to the debug menu
             m_DebugDisplaySettings.data.msaaSamples = m_MSAASamples;
 
+#if ENABLE_VIRTUALTEXTURES
+            Debug.Log("Scriptable renderpipeline VT enabled");
```

Comment:
Don't write log in the console (only for debugging), we must have nothing in the log when opening a project